reagurk/public/index.html

```diff
@@ -21,6 +21,23 @@
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <title>Gurkult</title>
+    <meta charSet="utf-8" />
+    <title>Gurkult | Open Source Community</title>
+    <meta name="description" content="The home for the Gurkans" />
+
+    <meta property="og:title" content="Gurkult | Open Source Community" />
+    <meta property="og:type" content="article" />
+    <meta property="og:url" content="https://www.gurkult.com/" />
+    <meta property="og:image" content="%PUBLIC_URL%/logo.png" />
+    <meta property="og:description" content="Learn open source by contributing to real world projects. Be a part of an inclusive, international community of developers." />
+
+    <meta name="twitter:card" content="website" />
+    <meta name="twitter:title" content="website" />
```

Comment:
```suggestion
    <meta name="twitter:title" content="Gurkult | Open Source Community" />
```
This title should remain the page title.